app/views/patron_requests/new.html.erb

```diff
@@ -13,14 +13,50 @@
   <% end %>
 <% end %>
 
-<%= form_for(@request, data: { controller: 'patronRequest' }, html: { class: 'col-lg-8 accordion' }) do |f| %>
+<%= form_for(@request,
+             data: { controller: 'patronRequest' },
+             html: { class: 'col-lg-8 accordion' },
+             url: current_request.aeon_page? ? current_request.aeon_form_target : patron_requests_path(request: current_request),
+             method: current_request.finding_aid? ? 'GET' : 'POST') do |f| %>
   <% step_enum = Enumerator.new { |y| n = 0; loop { y << (n += 1) } } %>
   <% current_step = nil %>
   <% scan_or_pickup_step = nil %>
   <%= f.hidden_field :instance_hrid %>
   <%= f.hidden_field :origin_location_code %>
+  <%= f.hidden_field :aeon_system_id, name: 'SystemID', value: 'sul-requests'  %>
+  <%= f.hidden_field :aeon_request_form, name: 'WebRequestForm', value: 'GenericRequestMonograph' %>
+  <%= f.hidden_field :aeon_document_type, name: 'DocumentType', value: 'Monograph' %>
+  <%= f.hidden_field :aeon_site, name: 'Site', value: f.object.aeon_site %>
+  <%= f.hidden_field :aeon_location, name: 'Location', value: f.object.origin_location_code %>
+  <%= f.hidden_field :aeon_title, name: 'ItemTitle', value: f.object.bib_data.title %>
+  <%= f.hidden_field :aeon_author, name: 'ItemAuthor', value: f.object.bib_data.author %>
+  <%= f.hidden_field :aeon_date, name: 'ItemDate', value: f.object.bib_data.pub_date %>
+  <%= f.hidden_field :aeon_link, name: 'ItemInfo1', value: f.object.bib_data.view_url %>
```

Comment:
What do you think about wrapping all these aeon parameters (and the other item params below) in `if current_request.aeon_page?`? I don't think any of them will conflict with the parameters we're using, but they might be a little strange to see when trying to debug stuff?

Reply:
Sure, no harm in doing that. I'll change it

Reply:
Updated